### PR TITLE
[INT2-114] Add governance and operations 

### PR DESCRIPTION
**Pull request by CRIPU**
done :) thanks!

### CODE REVIEW

File: governance/governance-and-operations.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# ガバナンスとオペレーション
+
+2021年4月25日に[YIP-61: Governance 2.0](https://gov.yearn.finance/t/yip-61-governance-2-0/10460)が可決されて以来、Yearnは**制約付委譲**によって管理される**マルチDAO**構造への移行を開始しました。このアプローチにより、プロトコル開発が官僚制度によって硬直化することなく、十分なレベルの分権性を維持することができます。
```
Comment (DarkGhost7) [inline governance/governance-and-operations.md:3]: ```YIP-61: Governance 2.0```
Translate Governance

File: governance/governance-and-operations.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+|yFarm|トレジャリーの拡大をする|YFI Secret Admirers, YFI Secret Entrance|
+|yTx|トランザクションを書き込む|YFI Doers|
+|yOps|コントリビューターの調整|YFI Ops|
+
+各yチームには、YIP-61で定義された特定の意思決定権が与えられています。
+
+| yチーム | 権限 | 説明 |
+|-------|-------|-------------|
+|yGuard|緊急時の権限|攻撃やバグが発生した場合、直ちに介入してストラテジーやvaultsをシャットダウンまたはロールバックします。|
+|yBrain|戦略の管理|戦略の有効化、無効化、調整、維持をします。|
```
[the remark this answers is not in the document]
Comment (DarkGhost7) [reply governance/governance-and-operations.md:67]: same as the ones below if possible

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]

File: governance/governance-and-operations.md (hunk 1 above)
Comment (CRIPU) [reply governance/governance-and-operations.md]: ガバナンス
I fixed it! thank u !

File: governance/governance-and-operations.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+- **YFIホルダー**はプロトコルやプロトコルのガバナンス構造の変更に投票します。
+- **yチーム** はプロトコルの特定の側面や関連する運用に集中します。
+- **マルチシグ**メンバーは、チェーン上の決定を実行または拒否します。
+
+トークン保有者は、どのようなyチームが存在するか、誰がマルチシグの一員であるか、各グループの運営管理の限界について最終的な決定権を持ちます。"制約付委譲"という言葉は、トークン保有者がYearnを構築・管理する様々なグループに特定の権限を委譲することに由来しています。
+
+ガバナンスの簡略化された流れは次のようになります。
+
+    1. YFIホルダーがyTeamの作成、破壊、制限を定義する
```
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I fixed it sorry > <

File: governance/governance-and-operations.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- **yチーム** はプロトコルの特定の側面や関連する運用に集中します。
+- **マルチシグ**メンバーは、チェーン上の決定を実行または拒否します。
+
+トークン保有者は、どのようなyチームが存在するか、誰がマルチシグの一員であるか、各グループの運営管理の限界について最終的な決定権を持ちます。"制約付委譲"という言葉は、トークン保有者がYearnを構築・管理する様々なグループに特定の権限を委譲することに由来しています。
+
+ガバナンスの簡略化された流れは次のようになります。
+
+    1. YFIホルダーがyTeamの作成、破壊、制限を定義する 
+    2. yチームがyTxに決定事項を通知する 
+    3. yTx が委任されたトランザクションを作成し、それをマルチシグに送信する
```
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I translated it as トランザクション
thanks!

File: governance/governance-and-operations.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+![](https://i.imgur.com/IDysF5O.png)
+
+### トークン保有者
+
+[YFIトークン](https://docs.yearn.finance/governance/yfi)保有者の義務として、プロトコルを改善する提案を作成し、投票することになっています。
+
+| 提案 | 説明 |
+|-----------|--------------|
+|Yearn Improvement Proposal (YIP)|YFIホルダーに委任された権限、または議会の権限の範囲外の権限を実行する提案|
```
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I translated it as 改善プロポーザル thanks!

File: governance/governance-and-operations.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+具体的には、これらの提案により、トークン保有者は以下のような意思決定を行うことができます : 
+
+| 権限 | 説明 |
+|-------|-------------|
+|権限の管理|YFI保有者は、yTeamへの、またはyTeamからの個別の権限の創設、割り当て、取り消しについて投票することができます。|
+|YFIトークンのコントラクト変更|YFIをミントしたり、ミンティングキーをバーンしたりするなど、YFIトークンコントラクトとのやりとりはすべてYFIホルダーの管理下にあります。|
+|料金体系の設定|Yearnプロトコルでの標準的な料金体系を設定します。|
+|マルチシグナリングの変更|短期的にはマルチシグが重要な権限を持ち続けるため、YFIホルダーのみがその署名者を変更するための投票を行います。|
+|yチームの承認|どのチームが権限を委譲されているかを管理するために、Yチームを正式に批准または廃止します。|
+|yOps署名者の変更|yOpsが他のyチームの署名者を変更する力を持っているように、これはyOpsの署名者を変更する特別な力です。|
```
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I translated it as オペレーション
Thanks!

File: governance/governance-and-operations.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+|トレジャリーファンドの使用|トレジャリーの資金を利用できます。|
+|YIP権限|YFIホルダーは、まだ委任されていないものについてYIPを提案する権限を持ちます。|
+
+### yチーム
+
+各Yチームは、トークン保有者によって定義されたオブジェクティブな権限と個別の権限を持っています。さらにメンバーシッププールに分けることができます。メンバーシッププールは、包括的なチームと同様の目標に向かって活動するコントリビューターの個別グループです。さらに、1つのメンバーシッププールが複数のyチームに所属することも可能です。
+
+| yチーム | オブジェクティブ | メンバーシッププール |
+|-------|-----------|-----------------|
+|yGuard|vaultsを守る｜YFI Protocol Dev, YFI Strategists, YFI Mechanics, YFI Secret Admirers|
```
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I think it's better to keep that part in English.
how do you think?

File: governance/governance-and-operations.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (CRIPU) [reply governance/governance-and-operations.md]: I fixed these part as katakana
thanks!

**Pull request merged**